tests/test_other.py

```diff
@@ -10027,6 +10027,8 @@ def test_deps_info(self):
         cmd.append('-sOFFSCREENCANVAS_SUPPORT')
       if function.startswith('wgpu'):
         cmd.append('-sUSE_WEBGPU')
+      if function.startswith('__cxa_'):
+        cmd.append('-sDISABLE_EXCEPTION_CATCHING=0')
```

Comment:
```suggestion
        cmd.append('-fexceptions')
```

Reply:
Is that the preferred way to spell this these day?

Reply:
I think so. When there is a "normal" clang flag that is a perfect alias then we prefer that.